apps/coscrad-frontend/src/components/resources/terms/term-index.spec.tsx

```diff
@@ -16,16 +16,16 @@ import { renderResourceIndexPageForTest } from '../test-utils';
 const dummyTerms: ITermViewModel[] = [
     {
         id: '12',
-        contributions: ['Jane Doe'],
+        contributions: [],
     },
 
     {
         id: '13',
-        contributions: ['Jane Doe'],
+        contributions: [],
```

Comment:
I wonder if we want these to be non-empty for this test. A bigger question is whether or not we still need a unit test given the comprehensive Cypress test.